packages/@lwc/rollup-plugin/src/__tests__/resolver/resolver.spec.ts

```diff
@@ -161,4 +161,16 @@ describe('resolver', () => {
 
         expect(warnings).toHaveLength(0);
     });
+
+    it('should throw an error when import stylesheet file is missing', async () => {
+        try {
+            await rollup({
+                input: path.resolve(__dirname, 'fixtures/missing-css/missing-css.js'),
+                plugins: [lwc()],
+            });
+        } catch (error: any) {
+            expect(error.pluginCode).toBe(1004);
+            expect(error.filename).toContain('missing-css/stylesheet.css');
+        }
+    });
```

Comment:
As you mention, this is a potentially breaking change, because currently you can import any nonexistent CSS file with any filename from a `*.js` file and [it will work](https://stackblitz.com/edit/salesforce-lwc-wvsjuy?file=src%2Fmodules%2Fx%2Fapp%2Fapp.js,src%2Fmodules%2Fx%2Fapp%2Fapp.html&title=LWC%20playground):


```js
import stylesheet from './app.css'
import stylesheet2 from './completelyFakeName.css'

console.log(stylesheet, stylesheet2) // logs undefined, undefined
```

Reply:
Thanks on the example that might break existing code! I've followed your suggestion to make it into a warning instead of an error so this PR should have no observable changes anymore.